aws/b2b_personalization/db_secrets.yaml

```
@@ -0,0 +1,18 @@
+dev:
```

wow sops setup worked without any issues on my system :)  
we keep the file a bit more general. for example we can name it `secrets.yaml` and move it into the `dev` folder.
```suggestion
postgres_db:
```